daemon/src/model/cfd.rs

```diff
@@ -243,19 +239,34 @@ pub enum CfdState {
         cet_status: CetStatus,
     },
 
-    /// The CFD contract's refund transaction was published but it not final yet
-    MustRefund {
+    /// The CET was published on chain but is not final yet
+    ///
+    /// This state applies to taker and maker.
+    /// This state is needed, because otherwise the user does not get any feedback.
```

Comment:
Can we not extend `CetStatus` and get rid of this one?

Reply:
I went with the `Pending` state because that's what we do for other transactions. Would at least keep some consistency until we a better idea of how we want to refactor the model. Happy to tackle this in follow-up(s).